tests/functional/DatadirTest.php

```diff
@@ -103,4 +113,19 @@ private function dropDatamodel(): void
         } catch (ServerException $exception) {
         }
     }
+
+    private function prepaireData(string $getTmpFolder): void
+    {
+        $finder = new Finder();
+        $finder->directories()
+            ->sortByName()
+            ->in($getTmpFolder)
+            ->exclude(['in', 'out'])
+            ->depth(0)
+        ;
+        foreach ($finder as $item) {
+            $this->replaceCredentials($item->getPathname());
+            $this->runScript($item->getPathname());
```

Comment:
Ta inicializacia mi pride po stranke funkcnosti ok, ale nejasna co sa tyka kodu a adresarovej struktury.

Pouzivas tam adresar `base-table` ... no mne by na prvy pohlad nenapadlo, ze je to nejaka inicializacia, 
... nebolo by lepsie pouzit `setup`, `init`, ...?

Z toho nie je vobec jasne, ze `$item->getPathname()` je nejaky `config.json` 
... pridaj to prosim do `finder`, ze hladas iba subory s nazvom `config.json` ... alebo `*.json` 

Bolo by dobre ak by si tam pridal, ze hladas iba v jednom adresary, `base-table` ... idealne `setup`/ `init`. 

Ide mi o to, ze ked sa niekto pozrie do kodu o pol roka, tak nebude musel pracne hladat co to robi a preco to tam je ... (mozno by sa hodil aj kratky koment v kode, ze preco to tam je a co to riesi).

Reply:
v data/source adresaři mohou býtpodsložky s dalšíma konfiguracema které chci vytvořit před hlavním během testu - 1 adresář = 1 row (tabulka) na zpracování

`base-table` proto protože je to základní tabulka na které testuju vazby (můžu přejmenovat na něco jiného, ale setup/init to není)

`$item->getPathname()` není config ale adresář ve kterém je config umístěný

Reply:
`v data/source adresaři mohou býtpodsložky s dalšíma konfiguracema které chci vytvořit před hlavním během testu - 1 adresář = 1 row (tabulka) na zpracování` ... ok, tak tie podlozky by mali byt v adresary `setup` / `init` ... aby bolo jasne, ze to nepouziva priamo komponenta, ale iba testovacie prostredie.

`$item->getPathname() není config ale adresář ve kterém je config umístěný` ... ok mas pravdu, zle som to pochopil, .... vdaka za vysvetlenie ... mozes tam prosim nad to hodit komentar, ... neviem, napr `Init test environment - run setup configuration from this sub-folder`.

Reply:
>tak tie podlozky by mali byt v adresary setup / init

jj to by mohly... přehodím to tam

Reply:
je to přehozený